tests/network/vmi_networking.go

```diff
@@ -185,17 +185,16 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		})
 
 		Context("with propagated IP from a pod", func() {
-
-			DescribeTable("should be able to reach", func(op k8sv1.NodeSelectorOperator, hostNetwork bool) {
-				if op == k8sv1.NodeSelectorOpNotIn {
-					requirements.MultipleNodes()
-				}
+			DescribeTable("should be able to reach", func(hostNetwork bool) {
+				requirements.MultipleNodes()
```

Comment:
Not really needed as the remote job will just fail on timeout.
The only disadvantage is that it will be hard to understand why. But that can be mitigated by a proper message, hinting about this possibility.

Reply:
I recall we talked offline about this topic and you asked to change the tests framework so tests can spesify their requirements (such as multi node) and fail in case requirements doesnt meet.

Having there requirements checks enable failing faster with informative message.
Its possible to drop all that and just test the test fail due to the Job timeout, but it make the test burn CI time on the server VMI creation and actually running the Jobs that will never become ready.
Please let me know what you think, and desire which direction to take (drop requiem tents check vs. let the test fail due to job timeout).

Reply:
When I see the result, I see it will fail anyway.

The desired direction is to do unify the tests as the PR subjects declares.
Adding infrastructure and all this additions is a distraction that slows us down at the moment.

What I commented before still stands, I have not seen additional info to convince me otherwise. But you are welcome to post a follow up PR or discuss it further as a stand-alone topic.

Reply:
> It will eventually fail, the suggested approach enable failing immediately and reduce CI time.

Why do I need to care about CI time if the setup is broken?
Let the one who runs the test check it and not run the tests in general.

> Sometimes its necessary to introduce additional changes then the test code for a PR to make sense.
> The current state is that we have a programmatic skip in this test (which we need to remove anyways), the additional changes (requirements package) done to get rid of the skip because it wont fit this PR changes.

I do not understand what you said here, sorry.

But you did convinced me that you should not skip or fail, you should make sure nothing changes from what existed before. If you want to change the existing behavior, you will have to send a new PR and reason for it independently.
If you do not have an additional node, do not start a new job and do not assert connectivity to/from it.

Reply:
This commit is removed, existing skip remain as is.

I will post a PR to remove this skip altogether.